src/qililab/instruments/yokogawa/gs200.py

```diff
@@ -198,6 +198,7 @@ def setup(self, parameter: Parameter, value: float | str | bool, channel_id: int
     @Instrument.CheckDeviceInitialized
     def initial_setup(self):
         """Performs an initial setup."""
+        self.device.off()
         self.source_mode = self.settings.source_mode
```

Comment:
how is this related to the pr description `Implement platform.execute_qprogram() for quantum machines`?

Reply:
It's a bug we found with Juanjo on Yoko and decided to add it here for easiness since only Roccqet uses the Yoko. Will add an entry in changelog.